apps/meteor/server/methods/OEmbedCacheCleanup.ts

```diff
@@ -12,6 +12,13 @@ declare module '@rocket.chat/ui-contexts' {
 	}
 }
 
+export const executeClearOEmbedCache = async () => {
+	const date = new Date();
+	const expirationDays = settings.get<number>('API_EmbedCacheExpirationDays');
+	date.setDate(date.getDate() - expirationDays);
+	await OEmbedCache.removeBeforeDate(date);
```

Comment:
Is the await necessary? You may just return the promise as well and let the caller await for it (since it's already async)

Reply:
Good idea, done!